man/systemd.network.xml

```diff
@@ -579,6 +579,12 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>Vrf=</varname></term>
```

Comment:
Hmm, we don't strictly follow a rule on this, but I think this should be called VRF= rather than Vrf=...